modules/ding_seo/ding_seo.install

```diff
@@ -0,0 +1,6 @@
+<?php
+
+/**
+ * @file
+ * Ding SEO install file.
+ */
```

Comment:
The info file and install file is added in the main ding-seo branch, so just remove this before rebasing.